### ISSUE TITLE
Last letter gets popped from the search query when using the "Go to subreddit" function

### ISSUE THREAD

**Issue by Mechragone** [ccrama/Slide#3104]
After opening the menu and attempting to go to a specific subreddit using its name, the last letter of my input disappears causing the search to fail.

Slide version: 6.2.2
Android version: 9.0 (Pie)

**Comment by Alexendoo** [comment]
What keyboard are you using @MechJM?

**Comment by MechJM** [comment]
Swiftkey

**Comment by Alexendoo** [comment]
Thanks, if you find the time could you capture a video of the behaviour? I installed swiftkey to test myself but couldn't get it to happen

**Comment by MechJM** [comment]
I don't think it's possible to submit videos in GitHub

**Comment by Alexendoo** [comment]
Yeah I think they only support images, but you can upload to something like [Streamable](https://streamable.com/) and link to that

**Comment by MechJM** [comment]
[Here you go.](https://streamable.com/pfjl6)

**Comment by MechJM** [comment]
I should also mention that since the initial post, my phone was updated to Android 10 and Slide was also updated obviously.

**Comment by Tensounder54** [comment]
Hiya! Does this issues still occur for you since you updated? And if so, could you re-upload the video (it's been taken down for some reason) as well as possibly a bug report? It will help us to resolve the issue faster if we know what we're dealing with. Thanks!

**Comment by MechJM** [comment]
Just installed version 6.5 from the play store and while I can still reproduce the issue, I noticed something new (it's possible it's been like this from the start but I just didn't notice it before). The last letter only gets popped if one (or more) of the letters in the search query is capitalized.

**Comment by Tensounder54** [comment]
Oh that's very odd. Not sure what's causing that but it's something that can be looked into. Thanks fr the video though. That should really help us to work out's what's happening here.
